Microsoft.Toolkit.Uwp.SampleApp/SamplePages/Animations/Effects/EffectAnimations.bind

```diff
@@ -0,0 +1,60 @@
+<Page
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:extensions="using:Microsoft.Toolkit.Uwp.UI.Extensions"
+    xmlns:animations="using:Microsoft.Toolkit.Uwp.UI.Animations"
+    xmlns:behaviors="using:Microsoft.Toolkit.Uwp.UI.Behaviors"
+    xmlns:media="using:Microsoft.Toolkit.Uwp.UI.Media"
+    xmlns:Interactivity="using:Microsoft.Xaml.Interactivity"
+    xmlns:Interactions="using:Microsoft.Xaml.Interactions.Core"
+    mc:Ignorable="d">
+
+  <Grid>
+    <Grid.RowDefinitions>
+      <RowDefinition Height="*"/>
+      <RowDefinition Height="Auto"/>
+    </Grid.RowDefinitions>
+    <Grid Height="560" Width="1280">
+      <Image Source="/Assets/Photos/SpeedTripleAtristsPoint.jpg" Stretch="UniformToFill"/>
+      <TextBlock
+          Text="THIS IS A SAMPLE TEXT"
+          FontSize="52"
+          FontWeight="Bold"
+          TextWrapping="Wrap"
+          VerticalAlignment="Center"
+          HorizontalAlignment="Center"/>
```

Comment:
```suggestion
    <Grid>
      <Image Source="/Assets/Photos/SpeedTripleAtristsPoint.jpg" Stretch="UniformToFill"/>
      <TextBlock
          Text="THIS IS SAMPLE TEXT"
          FontSize="52"
          FontWeight="Bold"
          TextWrapping="Wrap"
          VerticalAlignment="Center"
          HorizontalAlignment="Center"/>
```

Think it's cool if we let the image fill the available space?

Also we should remove 'A' from the text to be a bit more grammatically correct.